.travis.yml

```diff
@@ -1,25 +1,33 @@
-# Documentation: http://docs.travis-ci.com/user/languages/julia/
+## Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 
 os:
   - linux
+  - osx
 
 julia:
   - 1.0
+  - 1.2
+  - 1.3
+  - nightly
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - julia: nightly  # development versions
+
+# avoid duplicate tests in PRs
+branches:
+  only:
+    - master
 
 notifications:
   email: false
 
-jobs:
-  include:
-    - stage: "Documentation"
-      julia: 1.0
-      os: linux
-      script:
-        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd()));
-                                               Pkg.instantiate();'
-        - julia --project=docs/ docs/make.jl
-      after_success: skip
```

Comment:
Why remove the building of the documentation? This does not make sense to me.

Reply:
This is for building the docs using Github actions (https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#GitHub-Actions-1). I know that we are not using that it in other JuliaReach repos yet.

Reply:
Then I suggest to first try it out.

Reply:
i'll use this in BernsteinExpansions.jl

Reply:
i mean this is used in many repositories now, we don't need to wait.. ?

Reply:
I do not see the problem in trying it out first.

Reply:
ok, i'll update here when i have used this setup in a project that i manage.

Reply:
the BernsteinExpansions.jl docs are now built using Github actions workflow (https://juliareach.github.io/BernsteinExpansions.jl/latest/)

Reply:
should we merge this PR?

Reply:
Merging.... i've used this setup now both in BernsteinExpansions and in RangeEnclosures, with docs built ok and codecov true, but note that i added `codecov: true`.

Do open an issue if you have further simplifications / considerations! So i update the other repos too.